README.md

```diff
@@ -16,6 +16,8 @@ This diagnostic measures your ability to create and use Ember Objects and
 1.  Install dependencies by running `npm install` and `bower install`.
 1.  Create and checkout a new branch, named `response`.
 1.  Follow the directions given in [diagnostic.js](./public/diagnostic.js).
+1.  All diagnostic responses are to be made in [diagnostic.js](./public/diagnostic.js).
+1.  Do not work inside the actual ember app in this repository.
```

Comment:
I'm inclined to agree but Jeff said to be more explicit. Maybe it would have combined the two into something more concise :)